hcert_spec.md

```diff
@@ -29,22 +29,21 @@ Versions of this specification follow [semantic versioning](semver.org) and cons
 In addition, there is an _edition_ version number used for publishing updates to the document itself which has no effect on the HCERT, such as correcting spelling, providing clarifications or addressing ambiguities, et cetera. Hence, the edition number is not indicated in the HCERT. The version numbers are expressed in the title page of the document using a _major.minor.edition_ format, where the three parts are separated by decimal dots.
 
 
-## Electronic Health Certificate
+## Electronic Health Certificate Container Format
 
-The Electronic Health Certificate (HCERT) is designed to provide a uniform and standardised vehicle for health certificates from different Issuers. The aim is to harmonise how these health certificates are represented, encoded and signed with the goal of facilitating interoperability, while protecting the holder’s personal integrity and minimise costs in implementation.
+The Electronic Health Certificate Container Format (HCERT) is designed to provide a uniform and standardised vehicle for health certificates from different Issuers. The aim is to harmonise how these health certificates are represented, encoded and signed with the goal of facilitating interoperability.
 
+### The European Digital Green Certificate (DGC)
 
-### Coordinated Data Structure
+Ability to read and interpret a DGC issued by any Issuer requires a common data structure and agreements of the significance of each data field of the payload. To facilitate such interoperability, a common coordinated data structure is defined through the use of a JSON schema which consitutes the framing of the DGC. The use of these elements is outside the scope of this specification, and is anticipated to be regulated by European Union law.
 
```

Comment:
I would not start an explanation of DGC with a rationale.

I would recommend to start with something like "The European Digital Green Certificate (DGC) is a ....." and then provide rationales etc.